featurebyte/config.py

```diff
@@ -96,6 +97,7 @@ class Profile(BaseModel):
     name: str
     api_url: AnyHttpUrl
     api_token: Optional[str]
+    ssl_verify: bool = False
```

Comment:
Should it default to False?

Reply:
was testing previously, now defaulted to True